apps/meteor/server/startup/migrations/v292.ts

```diff
@@ -0,0 +1,32 @@
+import type { AppSignatureManager } from '@rocket.chat/apps-engine/server/managers/AppSignatureManager';
+import type { IAppStorageItem } from '@rocket.chat/apps-engine/server/storage';
+
+import { Apps } from '../../../ee/server/apps';
+import type { AppRealStorage } from '../../../ee/server/apps/storage';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 292,
+	name: 'Add checksum signature to existing apps',
+	async up() {
+		Apps.initialize();
+
+		const sigMan = Apps.getManager()?.getSignatureManager() as AppSignatureManager;
```

Comment:
Can both migrations be combined into one? since they're doing basically the same 👀

Reply:
@d-gubert